e2e/protractor.conf.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
```

Comment:
the sample doesn't have any end-to-end tests, does it? If not, this can be removed.